packages/epo-widget-lib/src/widgets/GalaxySelector/Points/Points.tsx

```diff
@@ -0,0 +1,59 @@
+import { FunctionComponent } from "react";
+import { AccessorKey, AstroObject, AstroType } from "../GalaxySelector";
+import Point from "../Point";
+
+interface PointsProps {
+  data?: AstroObject[];
+  selectedData?: AstroObject[];
+  activeId?: AstroType;
+  xValueAccessor: AccessorKey;
+  yValueAccessor: AccessorKey;
+  xScale: (value: number) => number;
+  yScale: (value: number) => number;
+  className?: string;
+  color?: string;
+}
+
+const Points: FunctionComponent<PointsProps> = ({
+  data = [],
+  selectedData = [],
+  activeId,
+  className,
+  xScale,
+  yScale,
+  xValueAccessor,
+  yValueAccessor,
+  color: colorOverride,
+}) => {
+  return (
+    <g className={className} role="list">
```

Comment:
used `list` and `listitem` roles in combo with `tabIndex=0` to make this visible to screenreaders.